### ISSUE TITLE
pickle data was truncated

### ISSUE THREAD

**Issue by kir1to455** [comprna/CHEUI#33]
Hi,
When I use CHEUI model 1 to predict modifications in individual reads, I saw "pickle data was truncated" in the log file.
![image](https://github.com/comprna/CHEUI/assets/87375686/851df307-546e-4d43-a602-efc148c4e51c)
Has there been an error here or has its prediction been successful?

Best wishes,
Kirito

**Comment by EduEyras** [comment]
Hi Kirito,

Did you check whether you run out of disc space or RAM during your run?
I cc Favour, AJ, and Stefan, in case they have seen this before in a CHEUI
or similar analysis.

Best

E.

**Comment by pre-mRNA** [comment]
Hi, 

Most likely, there was an issue with your prepreocssing which causes this issue when trying to run model 1. 

What was the exit code for your preprocess command?

**Comment by kir1to455** [comment]
Hi,
When I running preprocessing step, if I use the C++version, the subsequent process will not make any errors, but when I use the Python version, the error "pickle data was truncated" will occur.

This may be the reason I changed the Python script, as I changed the "with open" to "with gzip.open". 
Because my eventalign signal file is particularly large, I don't want to decompress it.

Anyway, when I decompressed the eventalign signal file and preprocessed it using C++version, everything ran normally.

Best wishes,
Kirito
